components/Lightweight/LightweightComponents.tsx

```
@@ -0,0 +1,216 @@
+          viewBox="0 0 24 24"
+        >
+          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
+        </svg>
+      </button>
+
+      {isOpen && (
+        <div className={cn(
+          "absolute z-50 mt-1 w-full bg-background border rounded-md shadow-lg max-h-60 overflow-auto"
+        )}>
+          <div
+            role="listbox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)